features/admins/admin_works.feature

```diff
@@ -93,6 +93,10 @@ Feature: Admin Actions for Works and Bookmarks
       And I should see "Mature"
       And I should see "Admin-Added Relationship"
       And I should see "Admin-Added Character"
+     When I follow "Activities"
+     Then I should see "View Admin Activity"
+     When I visit the last activities item
+     Then I should see "No Archive Warnings Apply"
```

Comment:
I wasn't familiar with this page, and from the test alone, I wasn't very sure what information it should display, so I went and played around until I figured it out. Could you maybe change the `Then` to make it clearer that the page is expected to show the old data and not the new? Something like:

```
Then I should see "Old tags"
  And I should see "User-Added Fandom"
  And I should not see "Admin-Added Fandom"
```